core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/DriverConfig.scala

```diff
@@ -1,12 +1,13 @@
 package org.virtuslab.ideprobe.config
 
-import org.virtuslab.ideprobe.config.DriverConfig.LaunchParameters
+import org.virtuslab.ideprobe.config.DriverConfig.{LaunchParameters, ScreenConfig}
 
 import scala.concurrent.duration._
 
 case class DriverConfig(
     launch: LaunchParameters = LaunchParameters(),
     check: CheckConfig = CheckConfig(),
+    screen: ScreenConfig = ScreenConfig(),
```

Comment:
It's not clear that this corresponds to Xvfb mode only, this should be called just `screen` 🤔

Reply:
you mean change `screen` name to something like xvfbScreen ?

Reply:
yeah, it's xvfb-specific, `xvfbScreen` FTW

Reply:
Hence, kebab-case to be applied here as well... actually, pls open an issue to extract some common base class for all configs so that it doesn't need to be defined everywhere